lib/org/chromium/webidl/CanonicalizerRunner.js

```diff
@@ -0,0 +1,60 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+foam.CLASS({
+  package: 'org.chromium.webidl',
+  name: 'CanonicalizerRunner',
+  extends: 'org.chromium.webidl.PipelineRunner',
+
+  requires: ['org.chromium.webidl.Canonicalizer'],
+
+  constants: {
+    MISSING_PROPS: 'Missing required properties in PipelineMessage',
+  },
+
+  properties: [
+    {
+      class: 'Int',
+      documentation: `Amount of time in seconds to wait after the last input
+        for the same source before forwarding canonical results to next runner.
+        Default value is 5 minutes (300 seconds).`,
+      name: 'waitTime',
+      value: 300,
+    },
+    {
+      name: 'canonicalizer_',
+      factory: function() {
+        return this.Canonicalizer.create({
+          onDone: this.output.bind(this),
+          waitTime: this.waitTime,
```

Comment:
Could `waitTime` ever be changed? If not, this is probably fine. If so, we need a binding.